bot/discord/events/ready.js

```diff
@@ -50,7 +50,7 @@ module.exports = async (client) => {
             "text": "free servers be created!",
             "type": "WATCHING"
         }, {
-            "text": "over " + users.length + " happy clients",
```

Comment:
Users refers the the global variable, which stores the user cache. It's an array. Users as in clients registered on the DBH panel, as opposed to the Discord server count.